development-defaults.ini

```diff
@@ -1,7 +1,7 @@
 
 path = "/uber"
 hostname = "localhost"
-url_root = "https://localhost:4443"
+url_root = "https://localhost"
```

Comment:
Was there a particular reason to do this, or was it just for your system?

You can use or create a `development.ini` file in the same directory as this file - it lets you override these config options without constantly having to check them into the repo.